### ISSUE TITLE
On-screen-keyboard fails to trigger some shortcuts

### ISSUE THREAD

**Issue by LWinterberg** [audacity/audacity#2191]
**Describe the bug**
On-screen-keyboard commands fail to execute "some shortcuts" according to a user. I could so far reproduce it on menus in particular, but I'm waiting to hear back from the user if this happens more generally. 

**To Reproduce**
1. Open the On-screen-keyboard
2. Press Alt twice on the OSK to emulate a short Alt-tap. 
3. Observe: The File menu gets highlighted, and the various menus gain the underlines/quick access key things. 
4. Press E on the OSK to open the Edit menu, or Arrow-right on the OSK to switch to the edit menu
5. Observe: The menu doesn't open, the action instead is performed outside the menu (eg arrow-right moves the cursor, as if the menu never was selected.

**Expected behavior**
The OSK should do exactly what a normal keyboard would do. In this case, Alt-tap -> arrow-right would select the Edit menu; Alt-tap -> E would open the edit menu immediately. 

**Additional information (please complete the following information):**
 - OS: Windows 11
 - Version 3.1.3-beta at least

**Additional context**
(reported via YouTube)

**Comment by DavidBailes** [comment]
Tested on Windows 10, and also impossible to open any of the menus on the menu bar using the on-screen keyboard.
This is also the case with:
1. One of the wxWidget sample programs.
2. One of microsoft's programs from the "windows classic samples" collection.
3. One of the sysinternals programs.
So this appears to be a bug with the widgets which wxWidgets are using on Windows.
